euphonic/validate.py

```diff
@@ -103,7 +103,7 @@ def _replace_dim(expected_shape: Tuple[int, ...],
 
 
 def _ensure_contiguous_attrs(obj: object, required_attrs: List[str],
-                             opt_attrs: List[str] = []) -> None:
+                             opt_attrs: List[str]|None = None) -> None:
```

Comment:
Does it need to be a list or is a `Sequence` ok?
```suggestion
                             opt_attrs: Sequence[str] = ()) -> None:
```

Reply:
I think any Collection will do, actually.